app/models/document_type.rb

```diff
@@ -32,8 +31,13 @@ def managed_elsewhere_url
     Plek.new.external_url_for(managed_elsewhere.fetch("hostname")) + managed_elsewhere.fetch("path")
   end
 
-  def guidance_for(id)
-    @guidance.find { |guidance| guidance.id == id }
+  def guidance_exists?(field_id)
```

Comment:
This seems like a good idea, to avoid repeating parts of the key everywhere. I think it would be better-placed in a helper, since it's primarily a view concern, and the point of this card is remove such concerns from the model.

Reply:
I'll move this to `I18nHelper` (or whatever that helper becomes) 👍